src/views/virtualmachines/details/tabs/disk/modal/DeleteDiskModal.tsx

```diff
@@ -98,15 +98,15 @@ const DeleteDiskModal: React.FC<DeleteDiskModalProps> = ({ vm, volume, isOpen, o
       isOpen={isOpen}
       obj={updatedVirtualMachine}
       onSubmit={onSubmit}
-      headerText={t('Detach {{diskName}} disk', { diskName })}
+      headerText={t('Detach {{diskName}} disk?', { diskName })}
       submitBtnText={t('Detach')}
       submitBtnVariant={ButtonVariant.danger}
     >
       <Stack hasGutter>
         <StackItem>
           <Trans t={t}>
             Are you sure you want to detach <strong>{diskName} </strong>in namespace{' '}
```

Comment:
You could add an `action` prop to `DeleteResourceMessage`, rename it to something like `ConfirmResourceActionMsg`, and use it for the detach messages as well. Just a suggestion.